frontend/src/utils/omakase.js

```diff
@@ -0,0 +1,90 @@
+import { fetchAuthSession } from "aws-amplify/auth";
+import {
+  EventBridgeClient,
+  PutEventsCommand,
+} from "@aws-sdk/client-eventbridge";
+
+export async function executeExport(formData, editTimeranges, flows) {
+  const { credentials } = await fetchAuthSession();
+
+  const region =
+    //@ts-ignore
+    import.meta.env.VITE_APP_AWS_REGION;
```

Comment:
The constants file i have already imports this constant so i suggest using that instead of "import.meta...." here.
e.g. `import { AWS_REGION } from "@/constants";`